.github/workflows/nodejs.yml

```diff
@@ -56,11 +56,16 @@ jobs:
       matrix:
         # https://github.com/actions/setup-node/issues/27
         node-version: [8.17.0, 10.x, 12.x, 13.x]
-        # Windows tests are failing for whatever reason, so just do linux and mac for now
-        os: [ubuntu-latest, macOS-latest]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
     runs-on: ${{ matrix.os }}
 
     steps:
+      - name: Set git config
```

Comment:
got this from the azure config

https://github.com/facebook/jest/blob/7d5956101c7755d4201337bbee2070628ace59fd/.azure-pipelines.yml#L12-L20